src/components/GovernanceCard.tsx

```
@@ -414,16 +417,50 @@ const GovernanceCard: FC<IGovernanceProps> = (props) => {
+												key={beneficiary?.address}
+												assetId={beneficiary?.genralIndex || null}
+												requestedAmt={beneficiary?.amount.toString()}
+												className={classNames(className, 'flex')}
+												postId={onchainId ? Number(onchainId) : (onchainId as any)}
+												proposalCreatedAt={created_at as any}
+												timeline={timeline || []}
+												usedInPostPage
+											/>
+										);
+									})}
+								{beneficiaries && beneficiaries?.length > 1 && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)